lib/core/src/lib/i18n/en.json

```diff
@@ -193,7 +193,7 @@
     },
     "METADATA": {
       "BASIC": {
-        "HEADER": "Properties",
```

Comment:
Btw changing this to `General info` instead `Properties` will break some functionalities in ACA and ADW as some extensions are checking that, please be aware of that and carefully verify if everything works

Reply:
You are talking about some extension. Can you give a brief about this?

Reply:
We have changed only in properties panel-title and not in Tabs. Below you can check the screenshot.
![image](https://github.com/Alfresco/alfresco-ng2-components/assets/111955585/a0bc5691-3500-4bba-9d81-1abf47adaa44)
Once the ADF changes merge then we can able to see the 'General Info' title instead of 'Properties'.

Reply:
As I said this will break some functionality in ADW, did you even check the code for governance info drawer service? This has to be adjusted to the changes you did here.

Reply:
@MichalKinas
We have found 2 occurrences in the governance info-drawer service. If you talking specifically about the below method where 'Properties' are there. Can I replace it with 'General info'?
```
public setMetaDataAspect(state: 'Record' | 'Properties') {
        this.store.dispatch(new SetInfoDrawerMetadataAspectAction(state));
    }

    private resetStoreAspect() {
        const isValidReset = this.displayAspect !== 'Properties';
        if (isValidReset) {
            this.setMetaDataAspect('Properties');
        }
    }
```

If you talking about the 'declare record' functionality then, that is working fine as earlier. Here is the conversation for the same.
https://github.com/Alfresco/alfresco-ng2-components/pull/8837#discussion_r1341015847